islandora_image.info.yml

```diff
@@ -0,0 +1,5 @@
+name: islandora_image
+type: module
+description: Islandora Image
+core: 8.x
+package: islandora
```

Comment:
@whikloj done! do i have to message you when i push a new commit or do you automatically get notified?